media/whip_connection.go

```diff
@@ -77,13 +78,23 @@ type WHIPPeerConnection interface {
 	GetStats() webrtc.StatsReport
 }
 
+type PeerConnStats struct {
+	ID              string
+	BytesReceived   uint64
+	BytesSent       uint64
+	PacketsReceived uint32
+	PacketsSent     uint32
+}
+
 type TrackStats struct {
-	Kind webrtc.RTPCodecType
-	*stats.Stats
+	Kind        webrtc.RTPCodecType
+	Jitter      float64
+	PacketsLost int64
```

Comment:
Kind of strange this is an int64 but PacketsReceived/PacketsSent on the peerconnection are uint32 but I assume that's how the underlying types are defined